CHANGELOG-DCA.rst

```diff
@@ -2,6 +2,51 @@
 Release Notes
 =============
 
+.. _Release Notes_7.56.0:
+
+7.56.0
+======
+
+.. _Release Notes_7.56.0_Prelude:
+
+Prelude
+-------
+
+Released on: 2024-08-16
```

Comment:
just curious: what does this date correspond to?

Reply:
When the tag was made I believe but since we had a few issues for this release the gap is noticeable indeed

Reply:
Ah makes sense, thanks!